docs/ldml/tr35-dates.md

```diff
@@ -2365,6 +2365,314 @@ The meaning of symbol fields should be easy to determine; the problem is determi
+
+##### <a name="Semantic_Skeleton_Length" href="#Semantic_Skeleton_Length">Length</a>
+
+**Required Fields: Year, Month, Day, Weekday, Hour, or Zone**
+
+**Default Value: Medium**
+
+The _length_ determines how wide the fields should be rendered. There are three choices:
+
+1. **Long:** Much space is available. Fields are typically spelled-out. Example: January 1, 2000
+2. **Medium:** Space is limited, and spelled-out fields are desired. Example: Jan. 1, 2000
+3. **Short:** Space is limited, and numeric fields are desired. Example: 1/1/2000
+
+Note: Unlike standard CLDR pattern and skeleton strings, there is only one length option for the whole semantic skeleton. This is based on the principle that developers ought to inform the library how much space is available and the context in which the date/time is being displayed, and translators ought to decide how to use that space. For example, it is possible for long month names and abbreviated weekday names to coexist, but that should be a translator decision, not a developer decision. However, this option may be extended in the future to allow hinting at lengths for individual fields.
+
+Note: The locale or calendar may coerce the month length to be different than the skeleton length. For example, there is no numeric representation of months in the Hebrew calendar in English, so spelled-out month names will be used in "en-u-ca-hebrew" even if the length is Short.
+
+Note: Additional lengths could be added in the future, such as "narrow" or "conversational".
+
+##### <a name="Semantic_Skeleton_Alignment" href="#Semantic_Skeleton_Alignment">Alignment</a>
+
+**Required Fields: Year, Month, Day, or Hour**
+
+**Default Value: Inline**
+
+The _alignment_ provides additional context that can be used for determining how to display certain fields, particularly numeric ones. There are two choices:
+
+1. **Inline:** The text will be displayed in a paragraph, label, heading, or similar context.
+2. **Column:** The text will be displayed vertically in a column-like layout or similar context where similar rendered widths are preferred.
+
+Note: The most common behavior with "column" alignment is for implementations to render a minimum of two digits on impacted fields. For example, an implementation might render "01/01/2000" instead of "1/1/2000" in US English.
+
+##### <a name="Semantic_Skeleton_Year_Style" href="#Semantic_Skeleton_Year_Style">Year Style</a>
+
+**Required Field: Year**
+
+**Default Value: Auto**
+
+The _year style_ defines the level of precision to use when displaying the year. There are three choices:
+
+1. **Auto:** Display the year with full or partial precision, and display the era if needed to disambiguate the year, depending on locale, calendar, and length. Example: Jan. 1, '00
```

Comment:
Use ‘00 instead of '00; add another example esp for Auto. We need to make sure that people are not mislead into thinking that Auto is confined.
```suggestion
1. **Auto:** Display the year with full or partial precision, and display the era if needed to disambiguate the year, depending on locale, calendar, and length. Example: Jan. 1, ‘00 or 1 January 2000
```

Now that I think about it, it might be clearer with a little table for each of the style options, instead of a numbered list.

Reply:
I used a lot of tables, but I didn't here because there was too much inline text, which doesn't render very well with tables.